apps/balancer-tools/src/components/KPI.tsx

```diff
@@ -0,0 +1,8 @@
+export function KPI({ title, content }: { title: string; content: string }) {
```

Comment:
this is not relevant to the whole application -- #should move it to `/apr/(components)`